src/styles/App.css

```diff
@@ -0,0 +1,144 @@
+* {
```

Comment:
- [ ] The mobile view has not been implemented yet.  I understand this is a work in progress or a future update.

![image](https://github.com/tan12082001/Vehicle-Booking-App-Frontend/assets/57421823/633ed84d-2a65-4e89-bed6-6089d90cec15)

_tip:  If you feel you've completed 90% of the features, kindly say in your PR description what features will be left out to give a heads-up to the final evaluator and reviewers._

_tip 2:  If you declare in your description that the mobile view will be left out, I will still approve the project, however, I don't recommend leaving it undone for final evaluation because this is a huge feature.  The final evaluator will make the final decision._

_tip 3: If you decide to implement it, you could do so in a simple, but functional way in order to improve your chances (MVP)._

_tip 4:  When dealing with mobile views, the flex column is our best, simple, fast friend.  Just make sure the content fits the screen and the navbar does not get in the way_